static/app/utils/events.tsx

```diff
@@ -228,12 +228,34 @@ function hasTrace(event: Event) {
  * by ensuring that every inApp frame has a valid sourcemap
  */
 export function eventHasSourceMaps(event: Event) {
-  const inAppFrames = getInAppFrames(event);
+  const inAppFrames = getFrames(event, true);
 
   // the map field tells us if it's sourcemapped
   return inAppFrames.every(frame => !!frame.map);
 }
 
+/**
+ * Function to determine if an event has been symbolicated
+ * (rawStacktrace is backfilled by stacktrace processors after processing)
+ */
+export function eventIsSymbolicated(event: Event) {
+  return event.entries?.some(entry => {
+    return (
+      entry.type === EntryType.EXCEPTION &&
```

Comment:
Do we wanna track eg `MESSAGE` events that contain the `threads` field with a valid `stacktrace` as well? they also have the `raw_stacktrace` backfilled.

Reply:
good idea, I can add that 👍 but `MESSAGE` is rather a UI component here, so we'd have to look up for the `THREADS` type